src/modals/WithdrawModal.js

```diff
@@ -259,6 +261,10 @@ class WithdrawModal extends React.Component {
           this.props.theme,
           15
         );
+
+        if (this.state.notifyEnabled) {
+          window.open(`https://9000.hal.xyz/recipes/loopring-withdrawal-notification?useraddress=${this.props.dexAccount.account.address}`);
```

Comment:
Does it always open a new tab to hal.xyz?

Reply:
Yes, It does as soon as the user leave the checkbox marked.

We can easily set by default false or instead of opening a new page and show a checkbox replace it with a link in the Withdrawal modal "Click here to enable notifications for withdrawal confirmation" similarly to what's already done for Ren.

Does it sound good for you?

Reply:
@emilianobonassi 

Thank you for your PR. However, I would not approve and merge the PR. 

The reasons are:
1. HAL is not wallet related. 
2. HAL is not a supported token on the exchange.
3. It's not a required feature for exchange.
4. The feature will redirect users to HAL page.